app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+
+    storage = {}
+
+    def wrapper(*args) -> Callable:
```

Comment:
You could use the `wraps` decorator from `functools` to preserve the wrapped function name and docstring.